README.md

```diff
@@ -56,8 +88,12 @@ env:
 
 The following environment variables are used to configure log collection.
```

Comment:
Thoughts on if we make this a section about all available environment variables, instead of just log collection?

Reply:
This will be handled by the documentation team.